client/send-video.cc

```diff
@@ -180,22 +180,22 @@ int main(int argc, char *argv[]) {
         return -1;
 
     // Determine required buffer size and allocate buffer
-    numBytes=avpicture_get_size(PIX_FMT_RGB24, pCodecCtx->width,
+    numBytes=avpicture_get_size(AV_PIX_FMT_RGB24, pCodecCtx->width,
```

Comment:
I guess this is an independent change, to make things work with a more current version of the av-lib ?